README.md

```diff
@@ -181,37 +199,67 @@ All of these libraries are already bundled within the Infusion image.
+finds on your system will be launched sequentially with each browser running the full Infusion test suite. The results
+will be returned in your terminal in the [TAP](https://testanything.org/) format. Once you have run `npm install`, you
+can use the command ``node node_modules/testem/testem.js launchers`` from the root of this repository to get a list of
+browsers that Testem can launch on your system.
+
+If you would like to debug individual tests or view the test summary in a browser, you can:
+
+1. Host the working directory, for example, using a command like the following from the root of the repository: `python -m SimpleHTTPServer 4102`
+2. Open the "rollup" file `tests/all-tests.html` that runs all tests in a browser.  Continuing the above example, you would load the URL `http://localhost:4102/tests/all-tests.html`.
+
+**Note:** Any browser launched will need to be focused and remain the active window. Some of the tests require focus,
+and will report errors if they are not focused.  If you want to run the tests consistently, your best option is to run
+the tests in a VM (see below).
 
 ### Run Tests Using Browsers Installed In a VM ###
 
-A [Fedora VM](https://github.com/idi-ops/packer-fedora) can be automatically created using tools provided by the [Prosperity4All Quality Infrastructure](https://github.com/GPII/qi-development-environments/). After meeting the [QI development VM requirements](https://github.com/GPII/qi-development-environments/#requirements) the ``vagrant up`` command can be used to launch a VM which will contain Testem and several browsers. Typing ``grunt tests`` will run the Infusion tests in the VM and the results will be displayed in your terminal.
+The tests in this package can be run within a virtual machine (VM).  The benefits of using a VM include the following:
+
+* Does not require testem to be installed on the host machine.
+* Allows other applications on the host machine to have focus while the tests are run.
+* Isolates the test run from issues specific to one operating system or machine.
 
-When this VM is first created Chrome and Firefox will be upgraded to the latest versions available in the Fedora and Google package repositories. The ``vagrant provision`` command can be used at a later time to trigger the browser upgrade and general VM provisioning mechanism.
+Before you can run tests within a VM, your machine will need to meet the
+[QI development VM requirements](https://github.com/GPII/qi-development-environments/#requirements).  Once you have that,
+a [Fedora VM](https://github.com/idi-ops/packer-fedora) can be automatically created using tools provided by the
+[Prosperity4All Quality Infrastructure](https://github.com/GPII/qi-development-environments/). The command
+``npm run test:vagrant`` (or ``yarn run test:vagrant``) will create the VM (if neeeded) and run the Infusion tests in
```

Comment:
Good idea, I added additional scripts and updated the docs.  I checked the previous universal PR to confirm that the wording roughly matches.